<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hato La Travesada</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../image/la_travesada_48_png.png</normaloff>../image/la_travesada_48_png.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(16, 57,4 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="txt_consola">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>741</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255,255,255);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_lanzar_server">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>506</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rekha</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #48dc6e, stop: 1 #39ae57
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #48dc6e, stop: 1 #50f379
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #48dc6e, stop: 1 #ddd
        );
    }
QPushButton:disabled {
    background-color: #6a737c;
    color: #333;
	}</string>
    </property>
    <property name="text">
     <string>Lanzar Servidor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_detener_server">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>506</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rekha</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #48dc6e, stop: 1 #39ae57
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #48dc6e, stop: 1 #50f379
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #48dc6e, stop: 1 #ddd
        );
    }
QPushButton:disabled {
    background-color: #6a737c;
    color: #333;
	}</string>
    </property>
    <property name="text">
     <string>Detener Servidor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_salir">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: rgb(127,0,0);
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #48dc6e, stop: 1 #39ae57
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #48dc6e, stop: 1 #50f379
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #48dc6e, stop: 1 #ddd
        );
    }
QPushButton:disabled {
    background-color: #6a737c;
    color: #333;
	}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_minimizar">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: rgb(127,0,0);
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #48dc6e, stop: 1 #39ae57
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #48dc6e, stop: 1 #50f379
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #48dc6e, stop: 1 #ddd
        );
    }
QPushButton:disabled {
    background-color: #6a737c;
    color: #333;
	}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_titulo">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rekha</family>
      <pointsize>31</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>Hato La Travesada</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_conexion_remota">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rekha</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>Conexi√≥n remota:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_enlinea">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: #48dc6e;
    padding: 5px;
    }

QPushButton:hover {
    background: #48dc6e;
    }</string>
    </property>
    <property name="text">
     <string>EN LINEA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_desconectado">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: rgb(255,255,255);;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: rgb(127,0,0);
    padding: 5px;
    }

QPushButton:hover {
    background: rgb(127,0,0);
    }</string>
    </property>
    <property name="text">
     <string>DESCONECTADO</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ip_local">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rekha</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
